src/main/app/src/tools/Utils.js

```diff
@@ -86,15 +86,23 @@ export class Parser {
 }
 
 export class OsoiteParser {
-  static getCoreAddress(katuosoite = '') {
+  static getCoreAddress(postitoimipaikka = '', osoite = '') {
     //Merkkejä ja välilyönnillä siitä erotettu numero, esim: Ratapiha 3, Hubert Hepolaisen Katu 888.
     //Mahdollinen jatke leikataan pois.
     const regexp = '^.+? \\d+';
-    const coreAddress = katuosoite.match(regexp);
+    const fullAddress = [postitoimipaikka, osoite].join(' ');
+    const withoutNumber = fullAddress
+      .split(' ')
+      .filter((s) => isNaN(s))
+      .join(' ');
+    const coreAddress = fullAddress.match(regexp);
     if (coreAddress === null) {
-      console.log('Warning: returning null for core address, input: ' + katuosoite);
+      console.log('Warning: returning null for core address, input: ' + fullAddress);
```

Comment:
Tämä voisi olla console.warn, console.logeja ei minusta saisi päätyä ollenkaan tuotantoon. Näitä konsolilogituksia voisi myös yrittää ehdollistaa niin että tuotantobuildeissa ei niitä välttämättä olisi ollenkaan.

Mutta käydään sitä joskus toiste läpi.